src/client/monitor/Monitor.tsx

```diff
@@ -33,6 +33,11 @@ export function Monitor({fullScreen, requestFullScreen}: MonitorProps): ReactEle
 
   const [loaded, setLoaded] = useState(false)
   const [projects, setProjects] = useState<Projects>([])
+  const [monitorClassNames, setMonitorClassNames] = useState({
```

Comment:
You don't need to store all of this as state. You only really need to keep track of the inactive boolean. Also doing this means the fullscreen style will never get toggled, as the state is only initialised on the first render using the `fullscreen` boolean.